automated-comments/elixir/solution/use_module_doc.md

```diff
@@ -0,0 +1,29 @@
+<!--
+This error is raised when the submission lacks @moduledoc for the Module
+-->
+
+It is recommended to have a [`@moduledoc`][1] attribute to describe the intent
+of the Module.
+
+- Documentation is a [first-class citizen][2] in Elixir -- It is designed to be easy to read and write.
+- Markdown syntax is supported inside of documentation attributes `@moduledoc` and `@doc`
+- when used the interactive console, `iex`, you can display the documentation generated by the compiler with the `h` function.
+
+```shell
+iex> h is_binary
```

Comment:
The comment is about a missing `@moduledoc`, though it uses a function as example in IEx here.

Reply:
I was trying to give an example to show why they are important to include, but I can understand why this might seem confusing.

Reply:
I'll just remove it

Reply:
removed